# HG changeset patch
# User Ken Brown <kbrown@cornell.edu>
# Date 1425654503 18000
#      Fri Mar 06 10:08:23 2015 -0500
# Node ID ad1b90a12676777818a170ede793f34765dcffe6
# Parent  3ee52afa4969411b4b7d61b2232c9274bcd1bce4
* src/makemake.in (BASE_MODULES): Work around gcc problem on 64-bit Cygwin

diff -r 3ee52afa4969 -r ad1b90a12676 src/makemake.in
--- a/src/makemake.in	Fri Mar 06 09:41:46 2015 -0500
+++ b/src/makemake.in	Fri Mar 06 10:08:23 2015 -0500
@@ -658,6 +658,11 @@
      ;;
 esac
 
+# Temporary workaround for gcc problem on 64-bit Cygwin
+if [ "$HSYSOS" = cygwin ] && [ "$HSYS" = x86_64 ]; then
+  BASE_MODULES="i18n syscalls"
+fi
+
 if [ "${with_gmalloc}" = yes ]; then GMALLOC=gmalloc; else GMALLOC=""; fi
 
 if [ $CROSS = true ] ; then
